### PR TITLE
Prevent session extension in SystemJobsStore

### LINKED ISSUE #3587: Unwanted session extension on SystemJobsStore
## Expected Behavior

Polling system jobs should not extend the user session. Similar changes have been made in the following PRs:

- https://github.com/Graylog2/graylog2-server/pull/3574
- https://github.com/Graylog2/graylog2-server/pull/3583

## Current Behavior

The SystemJobStore is polling without using the special HTTP header.

## Your Environment

* Graylog Version: 2.3.0-SNAPSHOT bbeefdddf35ae6cb0b00b495e432fd538681455a

### PR DESCRIPTION
**Pull request by joschi**
Fixes #3587

### CODE REVIEW

**Comment by joschi** [comment]
This has to be back-ported to the `2.2` branch once it has been reviewed and merged.

File: graylog2-web-interface/src/stores/systemjobs/SystemJobsStore.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Reflux from 'reflux';
 import URLUtils from 'util/URLUtils';
 import ApiRoutes from 'routing/ApiRoutes';
 import fetch from 'logic/rest/FetchProvider';
+import fetchPeriodically from 'logic/rest/FetchProvider';
```
Comment (edmundoa) [inline graylog2-web-interface/src/stores/systemjobs/SystemJobsStore.js]: This is not doing what it should. It uses the default export from `FetchProvider`, which is the `fetch` function, so there is no difference with the original code, except that the function is locally using another name.
In order to fix the issue we should use a named import here `import { fetchPeriodically }...`  to use the right function.

File: graylog2-web-interface/src/stores/systemjobs/SystemJobsStore.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import Reflux from 'reflux';
 
 import URLUtils from 'util/URLUtils';
 import ApiRoutes from 'routing/ApiRoutes';
-import fetch from 'logic/rest/FetchProvider';
+import {fetch, fetchPeriodically} from 'logic/rest/FetchProvider';
```
Comment (edmundoa) [inline graylog2-web-interface/src/stores/systemjobs/SystemJobsStore.js]: In this case this is also not the recommended way of importing both functions (it breaks the [`import/named`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/named.md) eslint rule). We should use this instead:
`import fetch, { fetchPeriodically } from 'logic/rest/FetchProvider';`

Comment (joschi) [inline graylog2-web-interface/src/stores/systemjobs/SystemJobsStore.js]: WTF JavaScript…?!

**Pull request merged**